lib/internal/bootstrap_node.js

```diff
@@ -312,62 +298,46 @@
 
   function setupGlobalConsole() {
     const originalConsole = global.console;
-    let console;
+    const Module = NativeModule.require('module');
+    // Setup Node.js global.console
+    const wrappedConsole = NativeModule.require('console');
     Object.defineProperty(global, 'console', {
       configurable: true,
       enumerable: true,
-      get: function() {
-        if (!console) {
-          console = (originalConsole === undefined) ?
-            NativeModule.require('console') :
-            installInspectorConsole(originalConsole);
-        }
-        return console;
+      get() {
+        return wrappedConsole;
       }
     });
-    setupInspectorCommandLineAPI();
-  }
-
-  function installInspectorConsole(globalConsole) {
-    const wrappedConsole = NativeModule.require('console');
-    const inspector = process.binding('inspector');
-    if (!inspector.consoleCall) {
-      return wrappedConsole;
+    // Setup inspector command line API
+    const { addCommandLineAPI, consoleCall } = process.binding('inspector');
+    if (!consoleCall) {
+      return;
     }
+    const { makeRequireFunction } = NativeModule.require('internal/module');
+    const path = NativeModule.require('path');
+    const cwd = tryGetCwd(path);
+
+    const consoleAPIModule = new Module('<inspector console>');
+    consoleAPIModule.paths =
+        Module._nodeModulePaths(cwd).concat(Module.globalPaths);
+    addCommandLineAPI('require', makeRequireFunction(consoleAPIModule));
```

Comment:
This is frankly unrelated to `console` instantiation. I'd prefer it to be moved out of console bootstrapping.

Reply:
I combined them because it was the only function calling the inspector. But I split them and the inspector part is now separated.